src/SB.Abstraction/Client/PublisherClient.cs

```diff
@@ -12,25 +12,23 @@ public class PublisherClient : IPublisher
     {
         public AzureSB.ServiceBusClient client { get; set; }
         private AzureSB.ServiceBusSender sender;
-        private string queue;
-        public PublisherClient(AzureSB.ServiceBusClient client,string nameQueue= "")
+        private string topic;
+        private string subs;
+        public PublisherClient(AzureSB.ServiceBusClient client,string nameTopic= "",string nameSubscription = "")
```

Comment:
please consider using method overloading instead of optional parameters

Reply:
it's a good debate